### PR TITLE
Adição CSS Handles

### PR DESCRIPTION
**Pull request by pconeglian**
#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/tsconfig.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "extends": "@vtex/tsconfig",
   "compilerOptions": {
+    "jsx": "react",
```
Comment (kaisermann) [inline react/tsconfig.json:4]: Why this addition?

File: react/BankInvoiceSection.tsx [hunk 2]
```diff
@@ -50,10 +59,10 @@ const BankInvoiceSection: FC = () => {
           values={{ paymentSystemName }}
         />
       </header>
-      <div className="flex-l justify-between items-center mt6">
+      <div className={`${handles.barCodeFlexContainer} flex-l justify-between items-center mt6`}>
         {barCodeNumber && <BarCode barCodeNumber={barCodeNumber} />}
         {isURLValid && (
-          <div className={`mt5 mt0-l ${barCodeNumber ? 'ml5-l' : ''}`}>
+          <div className={`${handles.printBilletButtonWrapper} mt5 mt0-l ${barCodeNumber ? 'ml5-l' : ''}`}>
```
Comment (kaisermann) [inline react/BankInvoiceSection.tsx:65]: This handle was not added to the readme's list! Couldn't it be called just `printButtonWrapper`?

File: react/BankInvoiceSection.tsx [hunk 3]
```diff
@@ -50,10 +59,10 @@ const BankInvoiceSection: FC = () => {
           values={{ paymentSystemName }}
         />
       </header>
-      <div className="flex-l justify-between items-center mt6">
+      <div className={`${handles.barCodeFlexContainer} flex-l justify-between items-center mt6`}>
```
Comment (kaisermann) [inline react/BankInvoiceSection.tsx:62]: I'd advise against using CSS terms in handles names. What if someone wants to modify this `div` to be `display: block`? We'd have a flex container which is not an actual flex container. What do you think of `barCodeContainer`?

```suggestion
      <div className={`${handles.barCodeContainer} flex-l justify-between items-center mt6`}>
```

File: react/BankInvoiceSection.tsx [hunk 4]
```diff
@@ -37,6 +44,8 @@ const BankInvoiceSection: FC = () => {
     message: `"${printInvoice}"`,
   })
 
+  const handles = useCssHandles(CSS_HANDLES)
```
Comment (kaisermann) [inline react/BankInvoiceSection.tsx:47]: Hooks must be called before any kind of forking code. It should be moved to the top of the component. The linter is also catching the error 😉

**Comment by pconeglian** [comment]
> Hey, @pconeglian 👋
> 
> First of all, thanks for your contribution! Please don't ignore the pull request template, as it helps us get to review your pull request with much more information than just your code alone.

Hi @kaisermann if you refer to the version part, I was not able to check any of them.

> It also seems that we have some lint errors, as you can see [here](https://github.com/vtex-apps/order-placed/pull/118/files). Would you be able to fix them? Just make sure to install the project's dependencies and run a `yarn lint --fix; yarn format`.

Going yo take a look! Thank you!

File: react/BankInvoiceSection.tsx (hunk 2 above)
Comment (pconeglian) [reply react/BankInvoiceSection.tsx:65]: Indeed I forgot.

File: react/BankInvoiceSection.tsx (hunk 3 above)
Comment (pconeglian) [reply react/BankInvoiceSection.tsx:62]: I get what you mean, but "flex-l justify-between items-center mt6" where already in there, I just wanted a way to reference to that particular DIV.

**Comment by kaisermann** [comment]
Finishing this in #120

**Pull request merged**